1 table.sql

```diff
@@ -0,0 +1,68 @@
+use FlyAirDB
+go 
+drop table if exists dbo.FlyAir
+go 
+
```

Comment:
1. General formatting tip: No need to have all spaces between column name and data type and constraint.
2. It's always good to prefix constraints with the table name.

Reply:
1. Fixed it. I did it this way, beacuse I thought it has a better overview like this.
2. Fixed it